run_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -ux
```

Comment:
do you really need bash to print the trace of all your commands executed?

Reply:
just copy-pasted from other script. Anyway it may be useful